ckan/config/routing.py

```diff
@@ -358,8 +358,6 @@ def make_map():
     map.redirect('/user/', '/user')
     with SubMapper(map, controller='user') as m:
         m.connect('/user/edit', action='edit')
-        # Note: openid users have slashes in their ids, so need the wildcard
-        # in the route.
```

Comment:
Can we remove the wildcards in the routes that this comment was referring to? eg `'/user/followers/{id:.*}` -> `'/user/followers/{id}`. Other routes don't use them anyway, and a year from now we will we wondering why they are there.